src/ShowAll.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import FakeBookings from "./data/fakeBookings.json";
```

Comment:
If you look carefully at the instructions for Exercise 6, you'll see it says:

> load data from the `src/data/fakeBookings.json` file in the `<Bookings />` component and pass it as a prop to `<SearchResults />`

Here we're loading the json directly inside the component and not passing it as a prop. Could you fix this so that the data is passed from `Bookings` to this component?

Reply:
The same I did